apps/online_oidc_provider/claims.py

```diff
@@ -42,3 +42,8 @@ class Onlineweb4ScopeClaims(ScopeClaims):
 
     def scope_onlineweb4(self):
         return Onlineweb4Userinfo(self.user).oidc()
+
+    info_nibble = (_("Nibble"), _("Informasjon om dine kjøp og saldo på Nibble"))
```

Comment:
I don't think we want a translation of `Nibble`, if there is one, since it is our own name for it (that is what `_`-does, which is actually the `django.utils.translation.gettext`

Reply:
Well spotted! As mentioned earlier, this is heavily copy-pasted so I'm not quite certain of whether this code is at all necessary. Whether it is or not, I'll remove the translation-method.